bin/bank/pycbc_brute_bank

```diff
@@ -237,6 +237,16 @@ class TriangleBank(object):
                 return False
 
             hc = self[j]
+
+            # Defensive initialization if matches/indices are missing
+            if not hasattr(hc, 'matches'):
+                hc.matches = numpy.empty(len(self))
```

Comment:
This is ok, but only if it's not really getting to this point often. It shouldn't be though.